manifests/install.pp

```diff
@@ -59,22 +53,4 @@
       fail("The provided install method ${nomad::install_method} is invalid")
     }
   }
-
-  if $nomad::manage_user {
-    user { $nomad::user:
```

Comment:
I think that the URL install method still relies on managing the user. Perhaps you need to drop that altogether?

Reply:
i dont know anyone that runs nomad as non root

Reply:
It can help to mention it explicitly in the commit message. Now it wasn't clear to me that packages use the root user.